selfdrive/test/process_replay/regen_all.py

```diff
@@ -1,22 +1,29 @@
 #!/usr/bin/env python3
+import argparse
+import concurrent.futures
+from tqdm import tqdm
+
+from selfdrive.test.process_replay.helpers import OpenpilotPrefix
 from selfdrive.test.process_replay.regen import regen_and_save
 from selfdrive.test.process_replay.test_processes import original_segments as segments
 
-if __name__ == "__main__":
-  new_segments = []
-  for segment in segments:
+def regen_job(segment):
+  with OpenpilotPrefix():
```

Comment:
how does this work without setting different log directories? did you make sure the logs are valid?

Reply:
hmm, guess it could work if there's a delay between starting the loggerd instances, though it should still work if they're all started at the same time

Reply:
The segment of each job is unique so why would you need different directories? also, I ran the segments generated in parallel through latency logger and they produce similar looking graphs. Also looked at them in plotjuggler and I think they look valid, but of course I did not look at all datatypes for all routes.